docs/index.rst

```diff
@@ -15,8 +15,8 @@ More Examples
 .. toctree::
   :maxdepth: 2
 
-  usage/index
-  Official Google BigQuery How-to Guides <https://cloud.google.com/bigquery/docs/how-to>
+  magics
+  Official Google BigQuery Magics Guide <https://cloud.google.com/python/docs/reference/bigquery/latest/magics>
```

Comment:
This page is going to change. Likely we'll want to publish sphinx docs to googleapis.dev similar to https://googleapis.dev/python/pandas-gbq/latest/

That said, if you're looking for an equivalent to the usage guides, that's not it anyway.

https://cloud.google.com/bigquery/docs/visualize-jupyter is close, though that's a tutorial, not a usage guide.

Reply:
ACK, putting https://cloud.google.com/bigquery/docs/visualize-jupyter link for interim now until we have the sphinx docs ready.